components/using_components.rst

```diff
@@ -40,7 +40,7 @@ If you know you need a specific version of the library, add that to the command:
 
 .. code-block:: bash
 
-    $ composer require symfony/finder
+    $ composer require symfony/finder 2.6.2
```

Comment:
It seems that @ifdattic is right because the example is specifically about using exact dependency versions.